index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <header id="header">
+    <nav id="nav-bar">
+      <ul>
+
+        <li><a class="nav-link" href="#about">About</a></li>
+        <li><a class="nav-link" href="#contact">Contact</a></li>
+        <li><a class="nav-link" href="#skills">skills</a></li>
+
+      </ul>
+    </nav>
+    <img id="header-img" src="https://www.bing.com/th?id=OIP.FcUYoInKYogVky8OJn08lgHaLH&w=120&h=170&c=8&rs
+      =1&qlt=90&o=6&pid=3.1&rm=2" alt="nature image" </header>
+    <!--about section-->
+    <section id="about">
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Magnam dolorem
+        voluptatum dolorum consectetur amet eos voluptatibus obcaecati?
+        Earum, beatae amet. Officia, eveniet? Quo, amet. Explicabo.</p>
+    </section>
+
+    <!--contact section-->
+    <section id="contact">
+      <p>our phone number : 2628920934209230</p>
+      <p>facebook page : aster-abass foundation</p>
+
+    </section>
+    <!--skills section-->
+    <section id="skills">
+      <p>we possess the following skills</p>
+      <ul>
+        <li>
+          HTML and css
+        </li>
+        <li>
+          javascript
+        </li>
+        <li>
+          python
+        </li>
+      </ul>
+    </section>
+    <video id="video" width="320" height="240" controls>
+      <source src="movie.mp4" type="video/mp4">
+
+    </video>
+
+
```

Comment:
- [ ] You should review these unrequited spaces and explain why its important to remove them to a student.